### DISCUSSION TITLE
When will flutter_slidable 1.0.0 be published as a stable release in pub.dev?

### DISCUSSION THREAD

**Discussion by heda7** [letsar/flutter_slidable#214] [Q&A] [answered]
I need to use flutter_slidable package in my company project but I noticed that it's still on version 0.6.0 so I guess I shouldn't use it until it reaches 1.0.0 and I was wondering when could this be?

**Comment by letsar** [comment, marked as answer]
Hi @heda7, the 1.0.0 will soon be stable, you can use this version in the meanwhile: https://pub.dev/packages/flutter_slidable/versions/1.0.0-dev.9

**Reply by letsar** [reply]
It's now published 😅
